### PR TITLE
Feature/api network 1.0

### PR DESCRIPTION
**Pull request by loopDelicious**
@kaustavdm @tristandenyer please review 🦐

### CODE REVIEW

File: v6/postman/launching_postman/newbutton.md [hunk 1]
```diff
@@ -154,18 +154,17 @@ In the **Next steps** tab, see a list of suggested next steps to get the most ou
...
-1. To configure your monitor, select how often you want the monitor to run and enter the URL you want to monitor in "CONFIGURATION OPTIONS" section.
-1. Click the **Create** button. The **Success!** screen displays what Postman created for you and suggests next steps to consider to get the most out of the template.
+1. Click on the template you want. In this example we're using the "Intro to writing tests - with examples" template. Each template has a description that may list the values required to execute the requests.
+   [![api-network-new-template-details](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/api-network/api-network-new-template-details.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/api-network/api-network-new-template-details.png)
+1. Click the **Use this template** button to summon the **Customization options** screen. This screen may list additional elements that are associated with this template and configuration options. Postman saves the values you enter as environment variables. Make sure you fill all the fields as these values might be crucial for the template to work properly.
```
Comment (kaustavdm) [inline v6/postman/launching_postman/newbutton.md]: Counter should increment. All items in this list show `1.`

File: v6/postman_for_publishers/api_network/add_api_network.md [hunk 2]
```diff
@@ -2,20 +2,42 @@
 title: "Add APIs to API network"
 page_id: "add_api_network"
 warning: false
-
 ---
 
-The Postman API Network provides the most trustworthy and easy-to-use directory of public APIs available. Every API listed in this network includes a complete Postman collection, created by the API's publisher. Postman specifically designed the API network to onboard developers quickly and effectively.
+The **Postman API Network** has amassed the widest variety of public APIs for your browsing pleasure. Verified publishers are required to submit their complete Postman collections for review. Community contributors can also share their own solutions and projects as [templates](/docs/postman_for_publishers/postman_templates/add_templates/) to help the broader community.
```
Comment (kaustavdm) [inline v6/postman_for_publishers/api_network/add_api_network.md]: Should the first mention of "Postman API Network" link to `explore.postman.com`?

File: v6/postman/launching_postman/newbutton.md (hunk 1 above)
Comment (loopDelicious) [reply v6/postman/launching_postman/newbutton.md]: who made these linting rules? incrementing the counter requires more work for updates.

Comment (loopDelicious) [reply v6/postman/launching_postman/newbutton.md]: and my linter shows no squiggles

**Pull request merged**